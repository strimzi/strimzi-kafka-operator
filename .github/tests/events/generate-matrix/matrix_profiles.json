{
  "event_name": "workflow_dispatch", 
  "action": "workflow_dispatch",
  "inputs": {
    "pipelines": "",
    "profiles": "operators,brokers-and-security"
  },
  "repository": {
    "name": "ci-playground",
    "owner": { "login": "frawless" }
  }
}