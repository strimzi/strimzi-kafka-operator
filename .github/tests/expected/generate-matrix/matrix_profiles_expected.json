[
  {
    "agent": "oracle-vm-8cpu-32gb-x86-64",
    "arch": "amd64",
    "pipeline": "regression",
    "profile": "operators",
    "timeout": 480,
    "strimzi_feature_gates": "false",
    "strimzi_rbac_scope": "CLUSTER",
    "cluster_operator_install_type": "yaml",
    "parallel": 4,
    "jobName": "regression-operators-amd64"
  },
  {
    "agent": "oracle-vm-8cpu-32gb-arm64",
    "arch": "arm64",
    "pipeline": "regression",
    "profile": "operators",
    "timeout": 480,
    "strimzi_feature_gates": "false",
    "strimzi_rbac_scope": "CLUSTER",
    "cluster_operator_install_type": "yaml",
    "parallel": 4,
    "jobName": "regression-operators-arm64"
  },
  {
    "agent": "oracle-vm-8cpu-32gb-x86-64",
    "arch": "amd64",
    "pipeline": "regression",
    "profile": "brokers-and-security",
    "timeout": 480,
    "strimzi_feature_gates": "false",
    "strimzi_rbac_scope": "CLUSTER",
    "cluster_operator_install_type": "yaml",
    "parallel": 4,
    "jobName": "regression-brokers-and-security-amd64"
  },
  {
    "agent": "oracle-vm-8cpu-32gb-arm64",
    "arch": "arm64",
    "pipeline": "regression",
    "profile": "brokers-and-security",
    "timeout": 480,
    "strimzi_feature_gates": "false",
    "strimzi_rbac_scope": "CLUSTER",
    "cluster_operator_install_type": "yaml",
    "parallel": 4,
    "jobName": "regression-brokers-and-security-arm64"
  },
  {
    "agent": "oracle-vm-8cpu-32gb-x86-64",
    "arch": "amd64",
    "pipeline": "regression-fg",
    "profile": "operators",
    "timeout": 480,
    "strimzi_feature_gates": "+DummyFeatureGate",
    "strimzi_rbac_scope": "CLUSTER",
    "cluster_operator_install_type": "yaml",
    "parallel": 4,
    "jobName": "regression-fg-operators-amd64"
  },
  {
    "agent": "oracle-vm-8cpu-32gb-arm64",
    "arch": "arm64",
    "pipeline": "regression-fg",
    "profile": "operators",
    "timeout": 480,
    "strimzi_feature_gates": "+DummyFeatureGate",
    "strimzi_rbac_scope": "CLUSTER",
    "cluster_operator_install_type": "yaml",
    "parallel": 4,
    "jobName": "regression-fg-operators-arm64"
  },
  {
    "agent": "oracle-vm-8cpu-32gb-x86-64",
    "arch": "amd64",
    "pipeline": "regression-fg",
    "profile": "brokers-and-security",
    "timeout": 480,
    "strimzi_feature_gates": "+DummyFeatureGate",
    "strimzi_rbac_scope": "CLUSTER",
    "cluster_operator_install_type": "yaml",
    "parallel": 3,
    "jobName": "regression-fg-brokers-and-security-amd64"
  },
  {
    "agent": "oracle-vm-8cpu-32gb-arm64",
    "arch": "arm64",
    "pipeline": "regression-fg",
    "profile": "brokers-and-security",
    "timeout": 480,
    "strimzi_feature_gates": "+DummyFeatureGate",
    "strimzi_rbac_scope": "CLUSTER",
    "cluster_operator_install_type": "yaml",
    "parallel": 4,
    "jobName": "regression-fg-brokers-and-security-arm64"
  }
]